centinel/views.py

```diff
@@ -326,12 +326,17 @@ def get_system_status():
     doesn't reveal anything important (e.g. IP address, username, etc.).
     The list is shuffled each time so that numbers are randomly assigned.
 
+    Note: we don't display clients who have the dont_display column set to 1/True
+
     """
     clients = Client.query.all()
     random.shuffle(clients)
     results = []
     number = 0
     for client in clients:
+        # dont add clients who have asked not to be displayed
+        if client.dont_display:
+            continue
```

Comment:
> Is that a serious question?

Is _that_ a serious question? lol
I meant to ask if you tested this with the server that is currently running. You need to add an extra column to the PostgreSQL DB and set the value for all of the existing clients and make sure it doesn't break things.